easybuild/easyconfigs/l/lftp/lftp-4.7.8-GCC-5.4.0-2.26.eb

```diff
@@ -0,0 +1,50 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-90.html
+# Modified::  Robert Qiao <robert.qiao@adelaide.edu.au> 
+#             @ adelaide.edu.au/phoenix 
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'lftp'
+version = '4.7.8'
+
+homepage = 'http://lftp.yar.ru'
+description = """LFTP is a sophisticated ftp/http client, and a file transfer program supporting
+  a number of network protocols. Like BASH, it has job control and uses the readline library for
+  input. It has bookmarks, a built-in mirror command, and can transfer several files in parallel.
+  It was designed with reliability in mind."""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+
+sources = [SOURCE_TAR_BZ2]
+source_urls = [
+    'http://lftp.yar.ru/ftp/',
+    'http://lftp.yar.ru/ftp/old/',
+]
+
+
+dependencies = [
+    ('GnuTLS', '3.5.14', '', ('GCCcore', '5.4.0')),
+    ('gettext', '0.19.8'),
+    ('Autoconf', '2.69', '', True), 
+    ('libreadline', '7.0', '', ('GCCcore', '5.4.0')),
```

Comment:
I think you'll need to stick with `6.3` here, due to other dependencies already using that

There's no need to hard specify which toolchain to use here, `GCCcore/5.4.0` will be considered automatically by EasyBuild v3 if no matching easyconfig using `GCC/5.4.0-2.26` is found

Reply:
@boegel do we have a flag that can update easyconfig file to a new toolchain and correct file name correspondingly?

Reply:
I think you're looking for `--try-toolchain`?